### ISSUE TITLE
📱(app-desk) Make the mobile version (low priority)

### ISSUE THREAD

**Issue by AntoLC** [numerique-gouv/people#123]
## Feature Request

All the website is not yet designed for the mobile version, so it is a low priority task, but we can slowly think about it.

--- 
⚠️ Be sure that the design is validated with the team before starting this dev (Johann / Antoine / Anto / JB ?).

![image](https://github.com/numerique-gouv/people/assets/25994652/943fc062-4a15-4a6d-acef-91a728e0c630)
